packages/gatsby-plugin-google-tag-manager/src/gatsby-browser.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import type { WrapRootElementBrowserArgs } from 'gatsby'
+
+const GTMProvider = require('./src/pixel/index').default
+
+export const wrapRootElement = (
+  { element }: WrapRootElementBrowserArgs,
+  { gtmId }: Props
+) => (
+  <GTMProvider
+    gtmId={gtmId}
+    // Overwrites the default blacklist config
+    dataLayerConfig={[]}
```

Comment:
Please make it possible to pass this prop via plugin config and make the default our DEFAULT_DATALAYER_CONFIG

Reply:
Actually, you could do a spread of plugin props

Reply:
Done